commands_automator_api.py

```
@@ -0,0 +1,99 @@
+        return self.commands_automator_service.get_script_description(script_file)
+    
+    def load_commands_configuration(self):
+        return self.run_async_method(self.commands_automator_config.load_configuration_async)
+
+    def save_commands_configuration(self, config):
+        return self.run_async_method(self.commands_automator_config.save_configuration_async, config)
+
+    def run_async_method(self, async_method, *args, **kwargs):
+        try:
+            return asyncio.run(async_method(*args, **kwargs))
+        except Exception as e:
```

Ruff (0.12.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)